### ISSUE TITLE
Feature Request: Make it apparent that the "Host" field is a dropdown or has a history

### ISSUE THREAD

**Issue by ScottBeeson** [bp2008/pingtracer#33]
I think ideally this could just be a combobox control? I am not a programmer, but I am a prolific user of tools.  I had no idea you could click the Host label.  I assume the blue is supposed to indicate a "link"? This is not standard UX. A combobox with autocomplete functionality would be _much_ preferred.

![image](https://user-images.githubusercontent.com/108358/157903640-f8b43b4a-5027-4b52-821c-da7083371d20.png)

**Comment by bp2008** [comment]
Yeah the blue color was supposed to hint to the user that they could click it.  I know it isn't standard, or expected by anyone to work that way.  But at the time, the app was only for me and this meant I didn't need to re-layout the UI.  Kind of like how the "Prefer IPv4" checkbox is positioned on top of the border of the Graph Options section; I didn't need to re-layout the UI this way.

I've been trying not to sink any more time into PingTracer 1.x development.  It is not a very good foundation.  Tracing the route is embarrassingly slow, dns lookups are done synchronously which is also slow, it is pretty broken on linux with mono, the traced route has no way to update itself if your traffic starts taking a different route, a lot of more-advanced monitoring situations require multiple instances of the app, which both overwrite each other's host history ...  I could go on ... point is, to fix all these things would require rewriting almost everything.

So some time ago I began designing version 2 that would solve all these problems.  But I stalled after getting this far.  Too many other demands on my time.

![image](https://user-images.githubusercontent.com/5639911/157905437-097a233f-681e-4c7c-a350-bd919e8e1da0.png)

I'm not sure how intuitive this graphing method is anyway.  The idea is, ping response time is represented by color.  The left edge of a graph is the first host on the route, the right edge is the last host, and it fills in like a waterfall so the newest path trace is drawn across the top, and when a new row is added it pushes down the previous rows.  It was my intent to label each graph and make it so that clicking on any of the waterfall graphs would bring up a different view where each host is labeled and you could pan to see all the available history (much like the classic pingtracer 1.x UI).

It was also my goal that you'd be able to monitor another process, so for example if you were having trouble with a multiplayer game, you could select that game's process and PingTracer2 would find out what remote hosts the game was communicating with in realtime and create graphs for each host.  I think this capability would be incredibly useful, but I could find no existing tool that does anything like it.

![image](https://user-images.githubusercontent.com/5639911/157909738-69fb0700-f9c0-4fcc-9523-aea40d72744e.png)

**Comment by ScottBeeson** [comment]
Very cool. The processes feature would be killer.
